remote-api/src/test/java/org/alfresco/rest/api/tests/NodeFolderSizeApiTest.java

```diff
@@ -0,0 +1,181 @@
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.api.tests;
+
+import org.alfresco.repo.security.authentication.AuthenticationUtil;
+import org.alfresco.rest.api.model.NodeTarget;
+import org.alfresco.rest.api.model.Site;
+import org.alfresco.rest.api.tests.client.HttpResponse;
+import org.alfresco.rest.api.tests.util.RestApiUtil;
+import org.alfresco.service.cmr.security.PermissionService;
+import org.alfresco.service.cmr.site.SiteVisibility;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.alfresco.rest.api.tests.util.RestApiUtil.toJsonAsStringNonNull;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * V1 REST API tests for Folder size
+ *
+ * @author Mohit Singh
+ */
+public class NodeFolderSizeApiTest extends AbstractBaseApiTest
+{
+
+    /**
+     * Private site of user two from network one.
+     */
+    private Site userOneN1Site;
+
+    protected PermissionService permissionService;
+
+    /**
+     * The logger
+     */
+    private static final Logger LOG = LoggerFactory.getLogger(NodeFolderSizeApiTest.class);
+
+    private String addToDocumentLibrary(Site testSite, String name, String nodeType)
+    {
+        String parentId;
+        try
+        {
+            parentId = getSiteContainerNodeId(testSite.getId(), "documentLibrary");
+            return createNode(parentId, name, nodeType, null).getId();
+        }
+        catch (Exception e)
+        {
+            LOG.error("Exception occured in NodeFolderSizeApiTest:addToDocumentLibrary {}", e.getMessage());
+        }
+        return null;
+    }
+
+    @Before
+    public void setup() throws Exception
+    {
+        super.setup();
+        permissionService = applicationContext.getBean("permissionService", PermissionService.class);
+    }
+
+    /**
+     * Test case for POST/calculateSize, which calculates Folder Size.
+     * <p>POST:</p>
+     * {@literal <host>:<port>/alfresco/api/<networkId>/public/alfresco/versions/1/nodes/<nodeId>/calculateSize}
+     */
+    @Test
+    public void testPostCalculateFolderSize() throws Exception
```

Comment:
add the order for the test and dependency on other test cases if they exist.

Reply:
Acknowledged.
But Junit <5.0, we don't have **priority** or **Order**.
And all relevant test cases are not dependent to each other.